hexa/databases/utils.py

```diff
@@ -1,28 +1,33 @@
 import psycopg2
 from psycopg2 import sql
 
-from hexa.plugins.connector_postgresql.models import Database
 from hexa.workspaces.models import Workspace
 
+from .api import get_db_server_credentials
 
-def get_workspace_database(workspace: Workspace):
-    return Database.objects.get(database="hexa-explore-demo")
 
+def get_workspace_database_url(workspace: Workspace):
+    credentials = get_db_server_credentials()
+    password = workspace.db_password
+    host = credentials["host"]
+    port = credentials["port"]
+
+    return f"postgresql://{workspace.slug}:{password}@{host}:{port}/{workspace.slug}"
 
-def get_database_definition(workspace: Workspace):
-    database = get_workspace_database(workspace)
 
-    with psycopg2.connect(database.url) as conn:
+def get_database_definition(workspace: Workspace):
+    url = get_workspace_database_url(workspace)
```

Comment:
I'd use the default database credentials to get db definition

Reply:
WDYT about it @pvanliefland ?

Reply:
I agree with @qgerome

Reply:
Okay works for me

Reply:
I don't think you implemented what @qgerome was suggesting - should't you use the default credentials?

Reply:
@cheikhgwane We can't give the permission to each role to query the default schema. Queries on `information_schema` have to be run with the user that creates the databases